### DISCUSSION TITLE
Port forwading from client

### DISCUSSION THREAD

**Discussion by ItsHass** [wg-easy/wg-easy#918] [Q&A]
Hi, 
Can anyone provide some guidance around how I would be able to port forward from the client, to the host, 

lets say nginx prot 80, on a client... and make it host at port 80 on the vpn host ...

How would this be possible ? iptables ? something could be made in the GUI ?

system is ubuntu minimal 22.04(cli).

thanks

**Comment by vithurshanselvarajah** [comment]
I think the wording of your question is causing some confustion. 

If you are looking to run a website on the same server as your wg-easy instance you will need to configure docker to use bridge mode. Docker by default will not allow it to access services on the host os. 

The easiest fix would be to add whatever you are trying to run as a additional docker image and configure it within a docker network to better security. (if you want to run apache webserver you can run it as a docker image and then create a docker network and provide wg-easy and apache ip's which your clients can then access)
